pkgs/applications/misc/dclock/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, haskellPackages }:
```

Comment:
Kindly use [nixfmt](https://search.nixos.org/packages?channel=24.11&from=0&size=50&sort=relevance&type=packages&query=nixfmt) to format your package

Reply:
Ran on most recent update